app/[userID]/page.tsx

```diff
@@ -0,0 +1,214 @@
+"use client";
+import { ThemeToggle } from "@/components/theme-trigger";
+import { Alert, AlertDescription, AlertTitle } from "@/components/ui/alert";
+import { Button } from "@/components/ui/button";
+import { MessageTypes, useServerConnection } from "@/hooks/useServerConnection";
+import { Info } from "lucide-react";
+
+import {
+    Card,
+    CardContent,
+    CardFooter,
+    CardHeader,
+    CardTitle,
+} from "@/components/ui/card";
+
+import { Verified } from "lucide-react";
+import Link from "next/link";
+import { redirect } from "next/navigation";
+import { useState } from "react";
+
+export default function Home({ params }: { params: { userID: string } }) {
+    const userId = params.userID;
+    const [userInfo, setUserInfo] = useState();
```

Comment:
Add a type parameter to userInfo, and use `null` instead of false

Like this (adjust as needed):

```typescript
import { MessageTypes, useServerConnection, type IdentificationMessage } from "@/hooks/useServerConnection";

const [userInfo, setUserInfo] = useState<IdentificationMessage["data"] | null>(null);
````